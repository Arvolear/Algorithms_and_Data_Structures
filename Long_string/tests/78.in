51615232046724566188161806602967238873289071520
30256358239427162160790619064209