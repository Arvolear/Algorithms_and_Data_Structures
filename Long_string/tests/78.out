51615232046724596444520046030129399663908135729
51615232046724535931803567175805078082670007311
1561688951416859735201678374554522347752202594029475174975562056451088873227680
1705930093710504